packages/experiments/src/components/Chiclet/examples/Chiclet.Footer.Example.tsx

```diff
@@ -17,6 +17,37 @@ export class FooterComponent extends React.Component<IFooterComponent, {}> {
   }
 }
 
+const footerStyle = mergeStyles({
+  height: 36,
+  paddingLeft: 11
+});
+const activitiesStyle = mergeStyles({
+  width: 184,
+  height: 16,
+  fontSize: 12,
+  fontWeight: 'normal',
+  fontStyle: 'normal',
+  fontStretch: 'normal',
+  lineHeight: 1.83,
+  letterSpacing: 'normal',
+  float: 'left',
+  paddingTop: 18,
+  paddingBottom: 12,
+  marginTop: 0
+});
+const actions = mergeStyles({
```

Comment:
Is there a reason the ones above have `Style` at the end of their name and these ones are just `actions` and `action`?

Reply:
Yes, the ones that have `Style` have the word before Style used somewhere else for a variable name, for consistency I am going to add `Style` to the end of all of them